tests/setupAfterEnv.js

```diff
@@ -2,3 +2,8 @@ import '@testing-library/jest-dom/extend-expect';
 import 'core-js/stable';
 import 'jest-axe/extend-expect';
 import 'regenerator-runtime/runtime';
+import { mswServer } from './mswServer';
+
+beforeAll(() => mswServer.listen());
```

Comment:
Won't this run everywhere, including places where this doesn't make sense, for example the react package?

Is it worth it, or is it useful anyway for every single test?

Reply:
Wild suggestion - we could have this isolated and in every test needed, we just import and use `mswServerSetup()` 🪄

Reply:
Could you check please?